src/VoteHandler.jsx

```diff
@@ -0,0 +1,58 @@
+import { useEffect, useState } from "react";
+import { upvoteArticleVotes, downvoteArticleVotes } from "../api";
+
+function VoteHandler(props){
+
+    const {articleId} = props;
+
+    const [voteCount, setVoteCount] = useState(0)
+
+    const [error, setError] = useState(null)
+
+    useEffect(() => {
+        setVoteCount((currVoteCount) => {
+            currVoteCount
+        })
+    }, [])
+
+
+    function handleUpvote(){
+        setVoteCount((currVoteCount) => {
+            currVoteCount + 1
+        })
+        setError(null)
+        upvoteArticleVotes(articleId)
+        .then((votes) => {
```

Comment:
We don't need this then block as we already fixed the render with the optimistic rendering on line 20